src/promptflow-evals/promptflow/evals/evaluate/_evaluate.py

```diff
@@ -376,14 +376,17 @@ def evaluate(
             columns=[col for col in evaluator_result_df.columns if col.startswith("inputs.")]
         )
 
-        # rename output columns
-        # Assuming after removing inputs columns, all columns are output columns
-        evaluator_result_df.rename(
-            columns={
-                col: "outputs." f"{evaluator_name}.{col.replace('outputs.', '')}" for col in evaluator_result_df.columns
-            },
-            inplace=True,
-        )
+        # If we use thread pool, the columns are already renamed.

```

Comment:
can we remove the rename logic from code_client instead?

Reply:
Fixed.